source/NetFramework/Server/Execution/Unbreakable/ApiPolicySetup.cs

```diff
@@ -74,7 +74,7 @@ public static ApiPolicy CreatePolicy() => ApiPolicy.SafeDefault()
                         t => t.Member(nameof(Conversions.FromCharAndCount), Allowed, new CountArgumentRewriter("Count"))
                               // Those need extra review
                               .Member(nameof(Conversions.ChangeType), Denied)
-                              .Member(nameof(Conversions.FallbackUserDefinedConversion), Denied)
+                              .Member("Conversions.FallbackUserDefinedConversion", Denied)

```

Comment:
This is not quite equivalent, `nameof` produces `"FallbackUserDefinedConversion"`, not the full name with the dot.
But that's fine, I am already looking to merge this change, so you don't need to adjust it.